client/src/components/frontend/ProjectTexts.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+
+export default class ProjectTexts extends Component {
```

Comment:
As above, this should be called GroupedTexts or something to that effect. The list is a list of texts grouped by tag. While that list is appearing in the context of the project, there's no need to codify that into the component name.